### ISSUE TITLE
Output Pyright version in Pylance log

### ISSUE THREAD

**Issue by debonte** [microsoft/pylance-release#5331]
Pylance currently outputs the hash of the latest Pyright commit that we have ingested:

```text
Pylance language server 2023.12.102 (pyright c93d9ebf) starting
```

It would be handy if it also included the Pyright version, acknowledging that the commit Pylance is based on might be between officially released versions.

**Comment by heejaechang** [comment]
This issue has been fixed in prerelease version 2023.12.103, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202312103-10-January-2024-PreRelease)

**Comment by Alexk1995** [comment]
gracias
